### PR TITLE
Implemented CalculatorVC

### PR DESCRIPTION
**Pull request by 0xifarouk**

### CODE REVIEW

File: DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    @IBOutlet var requestedThanawiyahTF: DesignableTF!
+    
+    var percantage = ["30%", "40%", "50%"]
+    var selectedPercentage: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createPercentagePicker()
+        createPickerToolbar()
+        ThanwiyahLabel.font = Identity.font(.custom(weight: .bold, size: 20))
```
Comment (cs4alhaider) [inline DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift:29]: Please extract  the ui related code to its own method

File: DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        percentagePicker.delegate = self
+        
+        requestedThanawiyahTF.inputView = percentagePicker
+        percentagePicker.backgroundColor = UIColor.clear
+        let blurEffect = UIBlurEffect(style: UIBlurEffect.Style.extraLight)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.frame = view.bounds
+        blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+    
+        // percentagePicker.addSubview(blurEffectView)
```
Comment (cs4alhaider) [inline DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift:69]: Please remove unused code

**Review by cs4alhaider — APPROVED** [review]

File: DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift (hunk 1 above)
Comment (0xifarouk) [reply DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift:29]: I will, it was just in initial implementation to test out things.

File: DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift (hunk 2 above)
Comment (0xifarouk) [reply DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift:69]: OK.

**Pull request merged**